File: Code_Logic/IBM_Cloud/openwhisk-python/CalculateFibonacci/code_logic/code_logic.py
def calculateFibonacci(number):
    if ((number == 1) | (number == 2)): return 1
    
    return calculateFibonacci(number - 1) + calculateFibonacci(number - 2)


def code():
    x = 38
    return calculateFibonacci(x)

File: Code_Logic/Amazon_Web_Services/aws-java-maven/CalculateFibonacci/code_logic/code_logic.java
package code_logic;

public class code_logic {

	static int number = 38;

	public static int calculate_Fibonacci (int x) {
		if (x == 1 || x == 2){
			return 1;
		}
		return (calculate_Fibonacci(x-1) + calculate_Fibonacci(x-2));
	}

	public static int code (){
		return calculate_Fibonacci(number);
	}

}
